frontend/src/utils/util.ts

```diff
@@ -19,3 +19,32 @@ export const downloadCSV = (filename: string, data: string) => {
   link.click()
   document.body.removeChild(link)
 }
+
+export interface BuildKiteEmoji {
+  name: string
+  url: string
+  aliases?: string[]
+}
+
+export const getEmojiUrls = (input: string, emojis: BuildKiteEmoji[]): string[] => {
+  const names = getEmojiNames(input)
+  return names.flatMap((name) => {
+    const emoji: BuildKiteEmoji = emojis.find((emoji) => emoji.name === name) as BuildKiteEmoji
```

Comment:
seems we don't need the cast statement here as we have specified the emojis as `BuildKiteEmoji []`

Reply:
test not work if there is no-type-statement

Reply:
ok then let's keep it here